test/path.jl

```diff
@@ -0,0 +1,5 @@
+@test @unix_only expanduser("~")[1] != ENV["HOME"]
```

Comment:
Ah didn't realise that, so I guess

    isabspath(expanduser("~")) == true

fails too.

Is there a reason this doesn't work on Windows? (Python's [`os.path.expanduser`](https://docs.python.org/2/library/os.path.html#os.path.expanduser) does.)